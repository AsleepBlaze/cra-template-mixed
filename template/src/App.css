html body #root {
  margin: 0;
  padding: 0;
  height: '100%';
  font-family: 'Gilroy';
  font-size: 16px;
  color: black;
  background: white;
}

button {
  border: 'none';
  cursor: 'pointer';
  color: black;
  margin: 0;
  padding: 0;
  font-family: 'Gilroy';
}

input {
  border: 'none';
  outline: 'none';
  padding: 0;
  font-size: 16px;
  color: black;
  background: 'transparent';
  font-family: 'Gilroy';
}

p {
  margin: 0;
  padding: 0;
}

a {
  color: black;
  text-decoration: 'none';
}

pre {
  font-family: 'inherit';
  white-space: 'pre-wrap';
  margin: 0;
}

h1,
h2,
h3,
h4,
h5,
h6 {
  margin: 0;
}
