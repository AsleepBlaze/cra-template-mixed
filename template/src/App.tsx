import React from 'react';
import { Provider } from 'react-redux';
import { BrowserRouter, Route, Routes } from 'react-router-dom';
import './App.css';
import store from './redux/store';
import Index from './components';
import axios from 'axios';
axios.defaults.baseURL = '/api/v1';

function App() {
  return (
    <Provider store={store}>
      <BrowserRouter>
        <Routes>
          <Route path="*" element={<Index />} />
        </Routes>
      </BrowserRouter>
    </Provider>
  );
}

export default App;
