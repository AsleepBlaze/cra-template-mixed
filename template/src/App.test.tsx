import { render } from '@testing-library/react';
import App from './App';

it('match app snapshot', async () => {
  const el = render(<App />);
  expect(el.container).toMatchSnapshot('app');
});
