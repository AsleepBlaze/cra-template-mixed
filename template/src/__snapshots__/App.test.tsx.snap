// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match app snapshot: app 1`] = `
<div>
  <header />
  <main />
  <footer />
</div>
`;
